Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -98,7 +95,13 @@ namespace PhysX
             }
         }
 
-        
+        //! Add the ground tot he scene via the test basis for tests where the ground is needed.
+        //! This function will only ever add one ground element to the scene.
+        void SetupGround(const AZ::Transform& floorTransform = DefaultFloorTransform)
+        {
+            m_floor = PhysX::TestUtils::AddStaticFloorToScene(m_sceneHandle, floorTransform);
+        }
+
         //! Function to remove the ground for tests where the ground can interfere with the intent of the test. 
         void RemoveGround()
```

Comment:
Are you using this any more? I think you can probably just not add the ground rather than add it and then remove it.

Reply:
Removed